imports/ui/components/detailLeft.jade

```diff
@@ -0,0 +1,9 @@
+template(name='detailLeft')
```

Comment:
I liked having the left and right panel templates in the same file as the parent.